vajram/vajram-codegen/src/main/java/com/flipkart/krystal/vajram/codegen/models/ParsedVajramData.java

```diff
@@ -51,10 +52,38 @@ public static Optional<ParsedVajramData> fromVajram(VajramInfo vajramInfo, Utils
             vajramInfo.responseType()));
   }
 
+  public static void handleDuplicateDependencyBySameResolverException(List<ExecutableElement> methods){
+    HashMap<String, HashMap<String,Boolean>> lookUpMap = new HashMap<>();
+    for(ExecutableElement method: methods){
+      String depName = method.getAnnotation(Resolve.class).depName();
+      String [] depInputs = method.getAnnotation(Resolve.class).depInputs();
+      for(String depinput : depInputs){
+        if (lookUpMap.get(depName)!=null && lookUpMap.get(depName).get(depinput)!=null){
+          String errorMessage =
+              "Two Resolver resolving same input (%s) for dependency name (%s)"
+                  .formatted(
+                      lookUpMap.get(depName).get(depinput),
+                      lookUpMap.get(depName));
+          throw new VajramValidationException(errorMessage);
```

Comment:
Add util.error() before throwing the exception so that the error is logged in console output.

Reply:
Okay